src/DependencyInjection/Configuration.php

```diff
@@ -124,9 +124,9 @@ private function addContextsSection(ArrayNodeDefinition $node)
                                         ->arrayNode('resizer_options')
                                             ->info('options directly pass to selected resizer. e.g. {use_crop: true, crop_gravity: center}')
                                             ->defaultValue([])
-                                            ->useAttributeAsKey('name')
-                                            ->prototype('scalar')
-                                            ->end()
+                                                ->useAttributeAsKey('name')
+                                                ->prototype('scalar')
+                                                ->end()
```

Comment:
this looks wrong, useAttributeAsKey and prototype were correct.

but `->end()` should be on the same "line" with `arrayNode`